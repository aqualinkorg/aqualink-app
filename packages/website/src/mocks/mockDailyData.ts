import { DailyData } from "../store/Reefs/types";

const now = new Date();
const minutesAgo = 5;
const dailyDataDate = new Date(
  now.getTime() - minutesAgo * 60000
).toISOString();

export const mockDailyData: DailyData = {
  id: 1,
  date: dailyDataDate,
  minBottomTemperature: 25,
  maxBottomTemperature: 27,
  avgBottomTemperature: 26,
  degreeHeatingDays: 29,
  surfaceTemperature: 36,
  satelliteTemperature: 20,
  minWaveHeight: 1,
  maxWaveHeight: 3,
  avgWaveHeight: 2,
  waveDirection: 136,
  wavePeriod: 15,
  minWindSpeed: 3,
  maxWindSpeed: 5,
  avgWindSpeed: 4,
  windDirection: 96,
  weeklyAlertLevel: 3,
};
