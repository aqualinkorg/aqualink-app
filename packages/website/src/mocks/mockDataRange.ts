import { TimeSeriesDataRange } from 'store/Sites/types';

export const mockDataRange: TimeSeriesDataRange = {
  bottomTemperature: [
    {
      type: 'hobo',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
  cholorophyllConcentration: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
  odoConcentration: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
  ph: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
  salinity: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
    {
      type: 'hui',
      depth: null,
      data: [
        {
          minDate: '2022-06-11T00:00:00.000Z',
          maxDate: '2022-06-14T23:45:00.000Z',
        },
      ],
    },
  ],
  turbidity: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
};
