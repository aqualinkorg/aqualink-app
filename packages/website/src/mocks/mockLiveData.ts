import { LiveData } from "../store/Sites/types";

const now = new Date();
const minutesAgo = 5;
const liveDataDate = new Date(now.getTime() - minutesAgo * 60000).toISOString();

export const mockLiveData: LiveData = {
  site: { id: 1 },
  bottomTemperature: {
    value: 25,
    timestamp: liveDataDate,
  },
  satelliteTemperature: {
    value: 26,
    timestamp: liveDataDate,
  },
  degreeHeatingDays: {
    value: 29,
    timestamp: liveDataDate,
  },
};
