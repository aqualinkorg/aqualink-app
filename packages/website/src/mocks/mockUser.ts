import { User } from "../store/User/types";

export const mockUser: User = {
  adminLevel: "default",
  administeredReefs: [],
  email: "mail@mail.com",
  firebaseUid: "RaNdOm StRiNg",
  fullName: "Mock User",
  organization: "Some organization",
  token: "RaNdOm StRiNg",
};
