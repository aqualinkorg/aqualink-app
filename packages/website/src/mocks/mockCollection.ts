import { CollectionDetails } from "../store/Collection/types";
import { mockReef } from "./mockReef";
import { mockUser } from "./mockUser";

export const mockCollection: CollectionDetails = {
  id: 1,
  isPublic: false,
  name: "Mock Collection",
  user: mockUser,
  reefs: [mockReef],
  reefIds: [1],
};
