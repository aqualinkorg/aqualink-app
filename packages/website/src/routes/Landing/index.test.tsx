import React from "react";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";
import configureStore from "redux-mock-store";

import LandingPage from ".";
import { mockUser } from "../../mocks/mockUser";
import { mockCollection } from "../../mocks/mockCollection";

const mockStore = configureStore([]);

describe("Landing Page", () => {
  let element: HTMLElement;
  beforeEach(() => {
    const store = mockStore({
      user: {
        userInfo: mockUser,
        loading: false,
        error: null,
      },
      collection: {
        details: mockCollection,
        loading: false,
        error: null,
      },
    });

    store.dispatch = jest.fn();

    element = render(
      <Provider store={store}>
        <Router>
          <LandingPage />
        </Router>
      </Provider>
    ).container;
  });

  it("should render with given state from Redux store", () => {
    expect(element).toMatchSnapshot();
  });
});
