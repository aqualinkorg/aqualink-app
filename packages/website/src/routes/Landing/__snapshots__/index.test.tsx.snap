// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page should render with given state from Redux store 1`] = `
<div>
  <mock-appbar
    classname="NavBar-appBar-6"
    color="primary"
    position="static"
  >
    <mock-toolbar
      classname="NavBar-toolbar-9"
    >
      <mock-drawer
        anchor="left"
        classes="[object Object]"
        open="false"
      >
        <mock-iconbutton
          style="align-self: flex-end; margin-right: 5px; margin-top: 5px; color: white;"
        >
          <clear />
        </mock-iconbutton>
        <mock-button
          classname="MenuDrawer-menuDrawerButton-19"
          component="[object Object]"
          to="/"
        >
          <mock-typography
            variant="h6"
          >
            HOME
          </mock-typography>
        </mock-button>
        <mock-button
          classname="MenuDrawer-menuDrawerButton-19"
          component="[object Object]"
          to="/map"
        >
          <mock-typography
            variant="h6"
          >
            MAP
          </mock-typography>
        </mock-button>
        <mock-button
          classname="MenuDrawer-menuDrawerButton-19"
          component="[object Object]"
          to="/buoy"
        >
          <mock-typography
            variant="h6"
          >
            BUOY
          </mock-typography>
        </mock-button>
        <mock-button
          classname="MenuDrawer-menuDrawerButton-19"
          component="[object Object]"
          to="/drones"
        >
          <mock-typography
            variant="h6"
          >
            DRONE
          </mock-typography>
        </mock-button>
        <mock-button
          classname="MenuDrawer-menuDrawerButton-19"
          component="[object Object]"
          to="/about"
        >
          <mock-typography
            variant="h6"
          >
            ABOUT
          </mock-typography>
        </mock-button>
        <mock-button
          classname="MenuDrawer-menuDrawerButton-19"
          component="[object Object]"
          to="/faq"
        >
          <mock-typography
            variant="h6"
          >
            FAQ
          </mock-typography>
        </mock-button>
        <mock-button
          classname="MenuDrawer-menuDrawerButton-19"
          component="[object Object]"
          to="/tracker"
        >
          <mock-typography
            variant="h6"
          >
            TRACK A HEATWAVE
          </mock-typography>
        </mock-button>
        <mock-button
          classname="MenuDrawer-menuDrawerButton-19"
          component="[object Object]"
          to="/register"
        >
          <mock-typography
            variant="h6"
          >
            REGISTER A SITE
          </mock-typography>
        </mock-button>
        <mock-box
          margintop="auto"
          padding="25px"
        >
          <mock-typography
            variant="subtitle1"
          >
            Aqualink's source code is licenced under MIT. Contribute!
          </mock-typography>
          <div
            class="MuiButtonGroup-root MuiButtonGroup-contained"
            role="group"
          >
            <mock-button
              classname="MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedContained MuiButtonGroup-groupedContainedHorizontal MuiButtonGroup-groupedContained MenuDrawer-contributeButton-18"
              color="default"
              disabled="false"
              disableelevation="false"
              disablefocusripple="false"
              disableripple="false"
              fullwidth="false"
              href="https://github.com/aqualinkorg/aqualink-app"
              size="medium"
              starticon="[object Object]"
              target="_blank"
              variant="contained"
            >
              GitHub
            </mock-button>
            <mock-button
              classname="MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedContained MuiButtonGroup-groupedContainedHorizontal MuiButtonGroup-groupedContained MenuDrawer-contributeButton-18"
              color="default"
              disabled="false"
              disableelevation="false"
              disablefocusripple="false"
              disableripple="false"
              fullwidth="false"
              href="https://ovio.org/project/aqualinkorg/aqualink-app"
              size="medium"
              target="_blank"
              variant="contained"
            >
              <img
                alt="Ovio Logo"
                src="ovio_logo.png"
              />
            </mock-button>
          </div>
        </mock-box>
      </mock-drawer>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-between"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 MuiGrid-grid-sm-2 MuiGrid-grid-md-2"
        >
          <mock-box
            alignitems="center"
            display="flex"
            flexwrap="nowrap"
          >
            <mock-iconbutton
              color="inherit"
              edge="start"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                />
              </svg>
            </mock-iconbutton>
            <mock-link
              classname="NavBar-navBarLink-7"
              href="/map"
            >
              <mock-typography
                color="textPrimary"
                variant="h4"
              >
                Aqua
              </mock-typography>
              <mock-typography
                style="color: rgb(138, 198, 222);"
                variant="h4"
              >
                link
              </mock-typography>
            </mock-link>
          </mock-box>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-flex-end MuiGrid-grid-xs-7 MuiGrid-grid-sm-4 MuiGrid-grid-md-8"
        >
          <mock-box
            alignitems="center"
            display="flex"
            flexwrap="nowrap"
          >
            Mock User
            <mock-iconbutton
              classname="NavBar-button-16"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root NavBar-expandIcon-15"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </mock-iconbutton>
            <mock-menu
              classname="NavBar-userMenu-12"
              keepmounted="true"
              menulistprops="[object Object]"
              open="false"
              popoverclasses="[object Object]"
            >
              <mock-divider
                classname="NavBar-userMenuDivider-11"
              />
              <mock-link
                classname="NavBar-menuItemLink-14"
                href="/dashboard"
              >
                <mock-menuitem
                  classname="NavBar-menuItem-13"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
                  >
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 5h4v6H5zm10 8h4v6h-4zM5 17h4v2H5zM15 5h4v2h-4z"
                          opacity=".3"
                        />
                        <path
                          d="M3 13h8V3H3v10zm2-8h4v6H5V5zm8 16h8V11h-8v10zm2-8h4v6h-4v-6zM13 3v6h8V3h-8zm6 4h-4V5h4v2zM3 21h8v-6H3v6zm2-4h4v2H5v-2z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      Dashboard
                    </div>
                  </div>
                </mock-menuitem>
              </mock-link>
              <mock-divider
                classname="NavBar-userMenuDivider-11"
              />
              <mock-menuitem
                classname="NavBar-menuItem-13"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M13 3h-2v10h2V3zm4.83 2.17l-1.42 1.42C17.99 7.86 19 9.81 19 12c0 3.87-3.13 7-7 7s-7-3.13-7-7c0-2.19 1.01-4.14 2.58-5.42L6.17 5.17C4.23 6.82 3 9.26 3 12c0 4.97 4.03 9 9 9s9-4.03 9-9c0-2.74-1.23-5.18-3.17-6.83z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item"
                  >
                    Logout
                  </div>
                </div>
              </mock-menuitem>
            </mock-menu>
          </mock-box>
        </div>
      </div>
    </mock-toolbar>
  </mock-appbar>
  <mock-dialog
    maxwidth="xs"
    open="false"
    scroll="body"
  >
    <mock-card>
      <div
        class="MuiCardHeader-root RegisterDialog-dialogHeader-21"
      >
        <div
          class="MuiCardHeader-content"
        >
          <mock-typography
            classname="MuiCardHeader-title"
            component="span"
            display="block"
            variant="h5"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-between"
            >
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <mock-typography
                    variant="h4"
                  >
                    Aqua
                  </mock-typography>
                  <mock-typography
                    classname="RegisterDialog-dialogHeaderSecondPart-22"
                    variant="h4"
                  >
                    link
                  </mock-typography>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <mock-iconbutton
                  classname="RegisterDialog-closeButton-20"
                  size="small"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                    />
                  </svg>
                </mock-iconbutton>
              </div>
            </div>
          </mock-typography>
        </div>
      </div>
      <mock-cardcontent>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
        >
          <div
            class="MuiGrid-root RegisterDialog-dialogContentTitle-23 MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10"
          >
            <div
              class="MuiGrid-root MuiGrid-item"
            >
              <mock-typography
                color="textSecondary"
                variant="h5"
              >
                Create an account
              </mock-typography>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10"
          >
            <form
              class="RegisterDialog-form-24"
            >
              <div
                class="MuiGrid-root RegisterDialog-textFieldWrapper-25 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-textfield
                  error="false"
                  fullwidth="true"
                  helpertext=""
                  id="firstName"
                  inputprops="[object Object]"
                  label="First Name"
                  name="firstName"
                  placeholder="First Name"
                  variant="outlined"
                />
              </div>
              <div
                class="MuiGrid-root RegisterDialog-textFieldWrapper-25 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-textfield
                  error="false"
                  fullwidth="true"
                  helpertext=""
                  id="lastName"
                  inputprops="[object Object]"
                  label="Last Name"
                  name="lastName"
                  placeholder="Last Name"
                  variant="outlined"
                />
              </div>
              <div
                class="MuiGrid-root RegisterDialog-textFieldWrapper-25 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-textfield
                  error="false"
                  fullwidth="true"
                  helpertext=""
                  id="organization"
                  inputprops="[object Object]"
                  label="Organization"
                  name="organization"
                  placeholder="Organization"
                  variant="outlined"
                />
              </div>
              <div
                class="MuiGrid-root RegisterDialog-textFieldWrapper-25 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-textfield
                  error="false"
                  fullwidth="true"
                  helpertext=""
                  id="emailAddress"
                  inputprops="[object Object]"
                  label="Email Address"
                  name="emailAddress"
                  placeholder="Email Address"
                  variant="outlined"
                />
              </div>
              <div
                class="MuiGrid-root RegisterDialog-textFieldWrapper-25 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-textfield
                  error="false"
                  fullwidth="true"
                  helpertext=""
                  id="password"
                  inputprops="[object Object]"
                  label="Password"
                  name="password"
                  placeholder="Password"
                  type="password"
                  variant="outlined"
                />
              </div>
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-between MuiGrid-grid-xs-12"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
                >
                  <mock-checkbox
                    checked="false"
                    classname="RegisterDialog-termsCheckbox-29"
                    color="primary"
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-sm-11"
                >
                  <mock-box>
                    <mock-typography
                      classname="RegisterDialog-formText-27"
                      color="textSecondary"
                      variant="subtitle1"
                    >
                      I have read the
                       
                      <a
                        class="RegisterDialog-termsLink-30"
                        href="/terms"
                      >
                        Terms and Conditions
                      </a>
                    </mock-typography>
                  </mock-box>
                </div>
              </div>
              <div
                class="MuiGrid-root RegisterDialog-button-28 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-button
                  color="primary"
                  disabled="true"
                  fullwidth="true"
                  size="large"
                  type="submit"
                  variant="contained"
                >
                  CREATE ACCOUNT
                </mock-button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-typography
                  classname="RegisterDialog-formText-27"
                  color="textSecondary"
                  variant="subtitle1"
                >
                  Have an account?
                   
                  <mock-button
                    color="primary"
                  >
                    SIGN IN
                  </mock-button>
                </mock-typography>
              </div>
            </form>
          </div>
        </div>
      </mock-cardcontent>
    </mock-card>
  </mock-dialog>
  <mock-dialog
    maxwidth="xs"
    open="false"
  >
    <mock-card
      elevation="0"
    >
      <div
        class="MuiCardHeader-root SignInDialog-dialogHeader-32"
      >
        <div
          class="MuiCardHeader-content"
        >
          <mock-typography
            classname="MuiCardHeader-title"
            component="span"
            display="block"
            variant="h5"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-between"
            >
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <mock-typography
                    variant="h4"
                  >
                    Aqua
                  </mock-typography>
                  <mock-typography
                    classname="SignInDialog-dialogHeaderSecondPart-33"
                    variant="h4"
                  >
                    link
                  </mock-typography>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <mock-iconbutton
                  classname="SignInDialog-closeButton-31"
                  size="small"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                    />
                  </svg>
                </mock-iconbutton>
              </div>
            </div>
          </mock-typography>
        </div>
      </div>
      <div
        class="MuiCollapse-container MuiCollapse-hidden"
        style="min-height: 0px;"
      >
        <div
          class="MuiCollapse-wrapper"
        >
          <div
            class="MuiCollapse-wrapperInner"
          >
            <div
              class="MuiPaper-root MuiAlert-root MuiAlert-standardSuccess MuiPaper-elevation0"
              role="alert"
            >
              <div
                class="MuiAlert-icon"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20,12A8,8 0 0,1 12,20A8,8 0 0,1 4,12A8,8 0 0,1 12,4C12.76,4 13.5,4.11 14.2, 4.31L15.77,2.74C14.61,2.26 13.34,2 12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0, 0 22,12M7.91,10.08L6.5,11.5L11,16L21,6L19.59,4.58L11,13.17L7.91,10.08Z"
                  />
                </svg>
              </div>
              <div
                class="MuiAlert-message"
              >
                Password reset email sent to .
              </div>
              <div
                class="MuiAlert-action"
              >
                <mock-iconbutton
                  aria-label="close"
                  color="inherit"
                  size="small"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                    />
                  </svg>
                </mock-iconbutton>
              </div>
            </div>
          </div>
        </div>
      </div>
      <mock-cardcontent>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
        >
          <div
            class="MuiGrid-root SignInDialog-dialogContentTitle-34 MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10"
          >
            <div
              class="MuiGrid-root MuiGrid-item"
            >
              <mock-typography
                color="textSecondary"
                variant="h5"
              >
                Sign In
              </mock-typography>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10"
          >
            <form
              class="SignInDialog-form-35"
            >
              <div
                class="MuiGrid-root SignInDialog-textFieldWrapper-36 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-textfield
                  error="false"
                  fullwidth="true"
                  helpertext=""
                  id="emailAddress"
                  inputprops="[object Object]"
                  label="Email Address"
                  name="emailAddress"
                  placeholder="Email Address"
                  variant="outlined"
                />
              </div>
              <div
                class="MuiGrid-root SignInDialog-textFieldWrapper-36 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-textfield
                  error="false"
                  fullwidth="true"
                  helpertext=""
                  id="password"
                  inputprops="[object Object]"
                  label="Password"
                  name="password"
                  placeholder="Password"
                  type="password"
                  variant="outlined"
                />
              </div>
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-button
                  classname="SignInDialog-forgotPasswordButton-40"
                >
                  <mock-typography
                    color="textSecondary"
                    variant="subtitle2"
                  >
                    Forgot your password?
                  </mock-typography>
                </mock-button>
              </div>
              <div
                class="MuiGrid-root SignInDialog-button-39 MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-button
                  color="primary"
                  fullwidth="true"
                  size="large"
                  type="submit"
                  variant="contained"
                >
                  SIGN IN
                </mock-button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12"
              >
                <mock-typography
                  classname="SignInDialog-formText-38"
                  color="textSecondary"
                  variant="subtitle1"
                >
                  Don't have an account?
                   
                  <mock-button
                    color="primary"
                  >
                    SIGN UP
                  </mock-button>
                </mock-typography>
              </div>
            </form>
          </div>
        </div>
      </mock-cardcontent>
    </mock-card>
  </mock-dialog>
  <div>
    <mock-box
      alignitems="top"
      classname="LandingPage-landingImage-1"
      display="flex"
    >
      <div
        class="MuiContainer-root LandingPage-container-2 MuiContainer-maxWidthLg"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-9"
        >
          <mock-box
            display="flex"
          >
            <mock-typography
              color="textPrimary"
              variant="h1"
            >
              Aqua
            </mock-typography>
            <mock-typography
              classname="LandingPage-aqualinkSecondPart-3"
              color="textPrimary"
              variant="h1"
            >
              link
            </mock-typography>
          </mock-box>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-sm-11 MuiGrid-grid-md-7"
        >
          <mock-box
            display="flex"
            mt="1.5rem"
          >
            <mock-typography
              color="textPrimary"
              variant="h1"
            >
              Temperature monitoring for marine ecosystems
            </mock-typography>
          </mock-box>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-sm-9 MuiGrid-grid-md-4"
        >
          <mock-box
            display="flex"
            mt="4rem"
          >
            <mock-typography
              color="textPrimary"
              variant="h4"
            >
              A tool for people on the front lines of ocean conservation
            </mock-typography>
          </mock-box>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <mock-box
            mt="2rem"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
            >
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <mock-button
                  classname="LandingPage-buttons-4"
                  color="primary"
                  component="[object Object]"
                  to="/map"
                  variant="contained"
                >
                  <mock-typography
                    variant="h5"
                  >
                    View The Map
                  </mock-typography>
                </mock-button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <mock-button
                  classname="LandingPage-buttons-4 LandingPage-registerButton-5"
                  color="primary"
                  component="[object Object]"
                  to="/register"
                  variant="outlined"
                >
                  <mock-typography
                    variant="h5"
                  >
                    Register Your Site
                  </mock-typography>
                </mock-button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <mock-button
                  classname="LandingPage-buttons-4 LandingPage-registerButton-5"
                  color="primary"
                  component="[object Object]"
                  to="/tracker"
                  variant="outlined"
                >
                  <mock-typography
                    variant="h5"
                  >
                    Track a Heatwave
                  </mock-typography>
                </mock-button>
              </div>
            </div>
          </mock-box>
        </div>
      </div>
    </mock-box>
  </div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <mock-box
      bgcolor="rgba(69, 76, 79, 0.05)"
      classname="Card-container-41"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
        >
          <mock-box
            classname="Card-textContainer-42"
          >
            <mock-box
              mb="1rem"
            >
              <mock-typography
                classname="Card-cardTitle-43"
                variant="h5"
              >
                Underwater temperature monitoring with a smart buoy
              </mock-typography>
            </mock-box>
            <mock-typography
              variant="h6"
            >
              There is some incredible existing technology that uses satellites to measure the ocean’s surface temperature from space. However, coral reefs and other important marine ecosystems are not always near the surface, and complex ocean dynamics can create a large temperature difference just a few meters down. To augment the satellite temperature, we partnered with Sofar Ocean to design a buoy that measures the temperature at the ocean floor. By building as many of these buoys as we can, and deploying them around the world, we can begin to build a dataset that helps in understanding where and when heat stress in the ocean will occur. By measuring, we can understand. And by understanding, we can begin to help.
            </mock-typography>
          </mock-box>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
        >
          <img
            class="MuiCardMedia-root Card-cardImage-44 MuiCardMedia-media MuiCardMedia-img"
            src="buoy.jpg"
          />
        </div>
      </div>
    </mock-box>
    <mock-box
      bgcolor="#ffffff"
      classname="Card-container-41"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-row-reverse MuiGrid-grid-xs-12"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
        >
          <mock-box
            classname="Card-textContainer-42"
          >
            <mock-box
              mb="1rem"
            >
              <mock-typography
                classname="Card-cardTitle-43"
                variant="h5"
              >
                Each site gets its own web page
              </mock-typography>
            </mock-box>
            <mock-typography
              variant="h6"
            >
              For each site, with or without a buoy, we report the surface temperature as measured from satellite, along with the historical max for that specific site, the alert level, and a time-series plot of the temperature data. If a buoy can be deployed, we will make sure it happens.
            </mock-typography>
          </mock-box>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
        >
          <img
            class="MuiCardMedia-root Card-cardImageScaleDown-45 MuiCardMedia-media MuiCardMedia-img"
            src="metrics.png"
          />
        </div>
      </div>
    </mock-box>
    <mock-box
      bgcolor="rgba(69, 76, 79, 0.05)"
      classname="Card-container-41"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
        >
          <mock-box
            classname="Card-textContainer-42"
          >
            <mock-box
              mb="1rem"
            >
              <mock-typography
                classname="Card-cardTitle-43"
                variant="h5"
              >
                Correlate survey observations with temperature changes
              </mock-typography>
            </mock-box>
            <mock-typography
              variant="h6"
            >
              Each site on the Aqualink web app has a detailed page outlining the satellite data, buoy data, heat stress alert level, as well as weather and a time history of the temperature. But perhaps most importantly for the site manager, it provides the ability to upload survey imagery and correlate it with all the other data. When a user uploads imagery, it provides a snapshot in time of the measured temperature with visual observations. By tagging the images, we are beginning to create a structured database that future machine learning algorithms can use to help predict heat stress events, such as coral bleaching. Adding surveys also connects the local ecosystem manager to a global network of scientists and engineers that are working everyday to protect these sensitive environments. The datasets are free and available to everyone.
            </mock-typography>
          </mock-box>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
        >
          <img
            class="MuiCardMedia-root Card-cardImageScaleDown-45 MuiCardMedia-media MuiCardMedia-img"
            src="survey-images.png"
          />
        </div>
      </div>
    </mock-box>
  </div>
  <mock-appbar
    classname="Footer-appBar-46"
    position="static"
  >
    <mock-toolbar>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10"
        >
          <mock-link
            classname="Footer-navBarLink-47"
            href="/map"
          >
            <mock-typography
              color="textPrimary"
              variant="h4"
            >
              Aqua
            </mock-typography>
            <mock-typography
              style="color: rgb(138, 198, 222);"
              variant="h4"
            >
              link
            </mock-typography>
          </mock-link>
        </div>
      </div>
    </mock-toolbar>
  </mock-appbar>
</div>
`;
